BinaryEncoding.md

```diff
@@ -142,33 +133,38 @@ A module may contain at most one import table section.
 #### Import entry
```

Comment:
Is there an ordering dependency that the Import Table must come after Signatures which it indexes?